build/FLExBridge.proj

```diff
@@ -98,6 +98,8 @@
 		<MakeDir Directories="$(RootDir)/output/$(Configuration)/"/>
 		<MakeDir Directories="$(RootDir)/output/$(Configuration)/net461/"/>
 		<MakeDir Directories="$(RootDir)/output/$(Configuration)/net461/localizations/"/>
+		<MakeDir Directories="$(RootDir)/output/DistFiles"/>
```

Comment:
I wonder - do we really need these directories here, or is it looking for them in this place because the binaries are now one level lower in `net461`? In the latter case it would probably be better to adjust the code to look in the new place.

Reply:
Right! Good thinking. I had a look for what's going on, but then found that at this point it is no longer necessary to copy DistFiles into output/. So that must have been fixed by earlier adjustments. 

I have changed the PR.